ARrehab/ARrehab/TileGrid.swift

```diff
@@ -73,7 +73,7 @@ class TileGrid {
      Adds the tile to the possibleTiles dict. and to the gridEntity
      */
     func generateOneTile(currentX: Float, currentZ: Float) {
-        let newTile = Tile(name: String(format: "Tile (%f,%f)", currentX, currentZ), x: currentX, z: currentZ, materials: [TileGrid.gridMaterial], adjustTranslation: true)
+        let newTile = Tile(name: String(format: "Tile (%f,%f)", currentX, currentZ), x: currentX, z: currentZ, materials: [TileGrid.gridMaterial], adjustTranslation: false)
         self.possibleTiles[newTile.coords] = newTile
         self.gridEntity.addChild(newTile)
```

Comment:
Can we have `gridEntity` be an intermediary Entity between the AnchorEntity and the tiles?

Reply:
on second thought its okay to just scale the anchorEntity. Lets clone the anchorEntity for the Minigame instead.

Reply:
> Can we have `gridEntity` be an intermediary Entity between the AnchorEntity and the tiles?

What do you mean by this? Grid Entity holds the tiles (which in turn hold the tile assets created by content team), and is attached to the surfaceAnchor (translated so centers line up). So in a sense it already is an intermediary between the surface anchor entity and the tiles

Reply:
GridEntity is an AnchorEntity so its at the root of the scene. At first I was hoping to have just (1) AnchorEntity and 2 sub entities: Board and Minigame.

However, I realize that its probably more consistent with the current implementation to have (2) AnchorEntities (but off of the same ARPlaneAnchor).

The purpose of having 2 such entities is so that we can disable the entire Minigame or Board hierarchy by changing one variable.